packages/worker/src/index.ts

```diff
@@ -11,7 +11,7 @@
 export { State, Worker, ActivityOptions, DataConverter, defaultDataConverter, RetryOptions, errors } from './worker';
 export { WorkerOptions, CompiledWorkerOptions } from './worker-options';
 export { ServerOptions, TLSConfig } from './server-options';
-export { Core } from './core';
+export { Core, CompiledCoreOptions, CoreOptions, TelemetryOptions } from './core';
```

Comment:
These options interfaces are used in `Core`'s functions but not exported. Perhaps not required for unused code detection, but a nice to export.

Reply:
👍